vuln/npm/419.json

```diff
@@ -0,0 +1,18 @@
+{
+  "id": 419,
+  "title": "Path Traversal",
+  "overview": "`superstatic` is vulnerable to path traversal on Windows",
```

Comment:
It's also vulnerable on certain Node.js versions on Linux. Not sure how to add that to the overview, and if it should be.

The only one that I am aware of is v9.9.0, but I have not tested all of them.

Reply:
I updated to   
```
"overview": "`superstatic` is vulnerable to path traversal on Windows and some Node.js versions",
```